<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSunColorDialog</class>
 <widget class="QDockWidget" name="CSunColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sun color</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Diffuse:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="NLQT::CColorEditWidget" name="diffuseWidget" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ambient</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="NLQT::CColorEditWidget" name="ambientWidget" native="true"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Specular:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="NLQT::CColorEditWidget" name="specularWidget" native="true"/>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLQT::CColorEditWidget</class>
   <extends>QWidget</extends>
   <header>color_edit_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
